src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="root-page">
+    <div class="calendar calendar--start-day_tue calendar--month-length_30">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
DON'T add numbers in HTML (you will do it using CSS)

Reply:
fixed